src/main/ui/app/component/AddOn.jsx

```diff
@@ -1,30 +1,39 @@
 import {Component} from "react";
 import {Link} from "react-router";
+import { ListGroupItem , Badge, Accordion , Panel , Media , Button } from 'react-bootstrap';
 
 export default class AddOn extends Component {
 
     render() {
         const addon = this.props.addon;
         const icon = addon.type === 'OWA' ? 'fa fa-2x fa-globe' : 'fa fa-2x fa-puzzle-piece';
         const link = this.props.version ? `/show/${addon.uid}?highlightVersion=${this.props.version}` : `/show/${addon.uid}`;
+	
+	const title = (
+ <div>{addon.name} <div className="pull-right">{addon.type}</div></div>
+);
+
         return (
                 <Link to={link}>
-                    <div className="media panel">
-                        <div className="media-left">
+                    <ListGroupItem className="mainlist">
+                   <Media>
+                        <Media.Left>
                             <i className={icon} aria-hidden="true"></i>
-                        </div>
-                        <div className="media-body">
-                            <h4 className="media-heading">
-                                {addon.name}
-                                <span className="badge">
-                                { this.props.version || "" }
-                            </span>
-                            </h4>
-                            {addon.description}
-                        </div>
-                    </div>
+                     	</Media.Left>
+                   <Media.Body> 
+                            
+    			
+		            
+    <Panel header={title} className="list-grp" eventKey="1">
```

Comment:
Formatting (clean up the extra blank lines, and also indent it like the other things)